luthier/config/src/tables.lua

```diff
@@ -136,19 +136,50 @@ M.physical = {
     Logical tables are defined in table M.logical. They map logical table
     names to their configuration:
 
-    * name - The name of the table
+    * The table is keyed on the name of the logical table
+    * category - the category in which the logical table belongs.
+        Defaults to 'GENERAL'
+    * longName - a longer name for descriptive uses.
         Defaults to the configuration's key
     * description - Brief documentation about the logical table.
         Defaults to the configuration's key
-    * metrics - A set of API metrics' name for this logical table, these 
-        should be a subset of the metrics configured in metrics.lua.
+    * type - [Mandatory] The nature of this logical table, used to build the table internally.
+        Currently supports:
+            * default - A simple set of rules just enough to describe a logical table correctly.
+                if 'default' is selected, the following fields will be used in the config:
+                    * metrics
+                    * dimensions
+                    * granularities
+                    * physicalTables
+                    * dateTimeZone
+                    * category
+                    * longName
+                    * retention
+                    * description
+                the following fields will be ignored:
+                    * apiFilters
+            * comprehensive - a list of rules to fully configure the logicalTable. This is not implemented yet.
```

Comment:
If this doesn't exist, I wouldn't mention it at all.

Reply:
OK. It is just a little bit strange to have only one item that we can default to in the next line.
But I will change it. I don't see the `comprehensive` which supposed to support apiFilters to be coded any time soon.

Reply:
This is a case of future proofing. Keep in mind that we're also providing support for people to add their own custom types. Furthermore, we may add additional logical table types in the future. In those cases, knowing what the defaulting behavior is going to be very important.